src/kibana/plugins/visualize/editor/sidebar.html

```diff
@@ -7,42 +7,68 @@
       {{ indexPattern.id }}
     </div>
 
-    <ul class="list-unstyled">
-      <!-- metrics -->
-      <vis-editor-agg-group ng-if="vis.type.schemas.metrics" group-name="metrics"></vis-editor-agg-group>
-
-      <!-- buckets -->
-      <vis-editor-agg-group ng-if="vis.type.schemas.buckets" group-name="buckets"></vis-editor-agg-group>
-
-      <!-- vis options -->
-      <vis-editor-vis-options
-        vis="vis"
-        >
-      </vis-editor-vis-options>
-
-      <!-- apply/discard -->
-    </ul>
-    <div class="vis-editor-sidebar-buttons">
-      <p
-        ng-if="visualizeEditor.softErrorCount() > 0"
-        class="text-center text-danger sidebar-item-text">
-        <i aria-hidden="true" class="fa fa-warning"></i>
-        {{ visualizeEditor.describeErrors() }}
-      </p>
-      <button
-        ng-disabled="!vis.dirty"
-        type="submit"
-        class="sidebar-item-button success">
-        Apply
-      </button>
-      <button
-        ng-disabled="!vis.dirty"
-        type="button"
-        ng-click="resetEditableVis()"
-        class="sidebar-item-button default">
-        Discard
-      </button>
+    <nav class="navbar navbar-default navbar-static-top subnav">
+      <div class="container-fluid">
+        <ul class="nav navbar-nav" ng-init="visConfigSection = vis.type.requiresSearch ? 'data' : 'options'">
+          <li ng-class="{active: visConfigSection == 'data'}" ng-show="vis.type.schemas.metrics">
+            <a class="navbar-link active" ng-click="visConfigSection='data'">Data</a>
```

Comment:
We do this all over the app, eg the time picker, the object editor. Its how bootstrap does it.  if we use another element we'll end up duplicating code for no good reason just to make some other element look like a navbar link. I don't see a reason to break the bootstrap conventions here

That we use buttons in the sub-navs is the reason hover styling is broken on all of them. We should probably change those.